domain/machine/state/state_test.go

```diff
@@ -124,3 +124,21 @@ VALUES (?, ?)
 	}
 	c.Assert(err, jc.ErrorIsNil)
 }
+
+func (s *stateSuite) TestListAllMachines(c *gc.C) {
+	err := s.state.UpsertMachine(context.Background(), "666")
```

Comment:
Can we rename `UpsertMachine` to `InsertMachine`.

Reply:
done in https://github.com/juju/juju/pull/17558